### PR TITLE
Fix missing whitespace in need-for-speed

### PR DESCRIPTION
**Pull request by angelikatyborska**
I solved the exercise honestly but still got accused of modifying the code because a single whitespace was missing in the analyzer 😄 

Screen Shot 2021-08-16 at 17 47 53

### CODE REVIEW

File: test/test_helper.exs [hunk 1]
```diff
@@ -1,2 +1,2 @@
 ExUnit.start()
-ExUnit.configure(exclude: [:pending, :external], trace: true)
+ExUnit.configure(exclude: [:pending, :external])
```
Comment (angelikatyborska) [inline test/test_helper.exs:2]: It's super annoying when the tests run with --trace, I can never find the failure then

Comment (jiegillet) [reply test/test_helper.exs:2]: Ah, yes, that is a quality of life improvement :)

**Review by jiegillet — APPROVED** [review]
Oh, that must have been hard to find. Sorry about that! 
The fix looks good :)

**Comment by angelikatyborska** [comment]
@exercism/maintainers-admin approve, pretty please?

**Review by iHiD — APPROVED** [review]

**Comment by iHiD** [comment]
This is great :)

**Comment by iHiD** [comment]
😉

**Pull request merged**